DataFormats/WrappedStdDictionaries/src/classes_def.xml

```diff
@@ -48,6 +48,7 @@
  <class name="edm::Wrapper<std::vector<unsigned short> >"/>
  <class name="edm::Wrapper<std::vector<std::pair<int,int>>>"/>
  <class name="edm::Wrapper<std::vector<std::pair<unsigned int,float>>>"/>
+ <class name="edm::Wrapper<std::vector<std::vector<float>>>"/>
```

Comment:
Does this dictionary still need to be added?

Reply:
no, but I thought it wouldnt do any harm to leave it? If its a problem I can remove it

Reply:
removed so so we move forward quickly